src/wp-includes/class-wp-dependencies.php

```diff
@@ -199,10 +199,31 @@ public function all_deps( $handles, $recursion = false, $group = false ) {
 				continue;
 			}
 
-			$keep_going = true;
+			$keep_going           = true;
+			$missing_dependencies = array();
+			if ( isset( $this->registered[ $handle ] ) && $this->registered[ $handle ]->deps ) {
+				$missing_dependencies = array_diff( $this->registered[ $handle ]->deps, array_keys( $this->registered ) );
+			}
 			if ( ! isset( $this->registered[ $handle ] ) ) {
 				$keep_going = false; // Item doesn't exist.
-			} elseif ( $this->registered[ $handle ]->deps && array_diff( $this->registered[ $handle ]->deps, array_keys( $this->registered ) ) ) {
+			} elseif ( $missing_dependencies ) {
+				// Prevent duplicate notices.
+				static $reported = array();
+
+				if ( ! isset( $reported[ $handle ] ) ) {
+					$reported[ $handle ] = true;
```

Comment:
What is the impact of this? Does it prevent a warning from being emitted during `wp_head` and then again in `wp_footer`?